src/index.html

```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
-                    </div>
-
-                    <h4 id="submit-submit"><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
-                    <div>
-                        <p>Upload your VCF file(s), metadata template and any associated data file(s) to your private FTP location.</p>
+                    <div class="submit-data-diagram">
+                        <div class="box green">
+                            <h3>📄 Prepare your data and metadata</h3>
+                            <ul>
+                                <li>Prepare valid VCF file(s), which can be validated prior to submission using the <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>.</li>
+                                <li>Complete a <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_template.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Please provide as much metadata as possible since this information is useful for downstream analysis and leads to increased dataset reuse. For reference, here is an <a href="https://www.ebi.ac.uk/eva/files/EVA_Submission_template.V1.1.5_mockup.xlsx" target="_blank"><b>example</b></a> of a completed metadata template.</li>
+                                <li>Please also note that the template requires the submitter to fill in some personal data, which will be used as described in our <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/eva-submission-service" target="_blank">privacy notice</a>.</li>
+                            </ul>
+                        </div>
+                        <div class="arrow"></div>
+                        <div class="row">
+                            <div class="box blue">
+                                <h3>💻 Submit via eva-sub-cli</h3>
+                                <p>This is our command-line tool for validating and uploading your data and metadata. For instructions on how to install and use <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>, please see our <a href="https://github.com/EBIvariation/eva-sub-cli/blob/main/README.md" target="_blank">documentation</a>.</p>
+                                <p>This method has faster turnaround times, but may be more suitable for more technical users.</p>
+                                <p>Before submitting data through eva-sub-cli, please make sure you have a <a href="https://www.ebi.ac.uk/ena/submit/webin/login" target="_blank">Webin</a> Account.</p>
+                            </div>
+                            <div class="or-label">OR</div>

```

Comment:
Should the OR be a bit larger? Maybe as large as the titles in the boxes

Reply:
updated